samples/bluetooth/unicast_audio_client/src/main.c

```diff
@@ -263,18 +263,6 @@ static void stream_released(struct bt_audio_stream *stream)
 	printk("Audio Stream %p released\n", stream);
 }
 
-static void stream_connected(struct bt_audio_stream *stream)
-{
-	printk("Audio Stream %p connected, start sending\n", stream);
-}
-
-static void stream_disconnected(struct bt_audio_stream *stream, uint8_t reason)
-{
-	printk("Audio Stream %p disconnected (reason 0x%02x)\n",
-	       stream, reason);
-	k_work_cancel_delayable(&audio_send_work);
```

Comment:
Don't we need this somewhere to release the timer?

Reply:
Nice catch! :)

Reply:
Is something being done for this?

Reply:
@cvinayak The `k_work_cancel_delayable(&audio_send_work);` was moved to the `stopped` callback instead